ui/apps/platform/src/types/config.proto.ts

```diff
@@ -44,6 +44,7 @@ export type DecommissionedClusterRetentionConfig = {
 
 export type ReportRetentionConfig = {
     historyRetentionDurationDays: number; // int32
+    downloadableReportGlobalRetentionBytes: number; // int64
```

Comment:
It looks like storage proto has uint32 (the existing property changed from original int32 yesterday)
https://github.com/stackrox/stackrox/blob/master/proto/storage/config.proto#L60C1-L62

Do you know anything about `downloadableReportRetentionDays` property also added in #7326

Reply:
Is that the source of truth? I see [swagger docs](https://github.com/stackrox/stackrox/blob/master/generated/api/v1/config_service.swagger.json#L308) show int64